src/encoded/schemas/file.json

```
@@ -224,6 +225,48 @@
             "description": "A flag to indicate whether this file is subject to restricted access",
             "type": "boolean"
         },
+        "s3_lifecycle_category": {
+            "title": "S3 Lifecycle Category",
+            "description": "The lifecycle category determines how long a file remains in a certain storage class.  If set to ignore, lifecycle management will have no effect on this file.",
+            "type": "string",
+            "suggested_enum": [
+                "short_term_access_long_term_archive",
+                "long_term_access_long_term_archive",
+                "long_term_access",
+                "short_term_access",
+                "long_term_archive",
+                "short_term_archive",
+                "no_storage",
+                "ignore"
+            ]
+        },
+        "s3_lifecycle_status": {
+            "title": "S3 Lifecycle Status",
+            "description": "Current S3 storage class of this object.",
```

Here, and elsewhere, we generally don't end descriptions with a period.
```suggestion
            "description": "Current S3 storage class of this object",
```